modules/vaos/app/models/ccra/referral_detail.rb

```diff
@@ -27,7 +27,8 @@ def initialize(attributes)
       @station_id = attributes[:station_id]
       @uuid = nil # Will be set by controller
       @has_appointments = attributes[:appointments].present?
-
+      # NOTE: appointment_type_id defaulted to 'ov' for phase 1 implementation
```

Comment:
What is OV? Maybe explain here.

Reply:
OV is the default appointment_type_id, not much to explain that's just what it needs to be set as as per Lee so that calls to EPS will work